jupyterhub/orm.py

```diff
@@ -343,7 +351,7 @@ def find(cls, db, name):
 class User(Base):
     """The User table
 
-    Each user has a single server,
+    Each user can have more than a single server,
```

Comment:
I think it may help future developers to be a bit more precise when using the word `servers` since it's a frequently used term in the JupyterHub world.

Here's one suggestion for revising the docstring:

```
Each user can have one or more single user notebook servers.

Each single user notebook server will have a unique token for authorization.
Therefore, a user with multiple notebook servers will have multiple tokens.

```